runtime_scan/pkg/provider/docker/helpers.go

```diff
@@ -0,0 +1,86 @@
+package docker
+
+import (
+	"fmt"
+	"github.com/openclarity/vmclarity/api/models"
+	"github.com/openclarity/vmclarity/runtime_scan/pkg/provider"
+	"os"
+	"strings"
+)
+
+func convertTags(tags map[string]string) *[]models.Tag {
+	ret := make([]models.Tag, 0, len(tags))
+	for key, val := range tags {
+		ret = append(ret, models.Tag{
+			Key:   key,
+			Value: val,
+		})
+	}
+	return &ret
+}
+
+func getScanName(config *provider.ScanJobConfig) (string, error) {
+	id, err := getAssetId(config)
+	if err != nil {
+		return "", fmt.Errorf("failed to get scan name. Provider=%s: %w", models.Docker, err)
+	}
+	return "scan_" + config.ScanID + "_asset_" + strings.Replace(id, ":", "_", -1), nil
+}
+
+func getScanConfigFileName(config *provider.ScanJobConfig) (string, error) {
```

Comment:
`scanConfigFileNameFor` is more descriptive

Reply:
using get* for consistency 👍